zone/client.cpp

```diff
@@ -3761,7 +3761,82 @@ void Client::Escape()
 	MessageString(Chat::Skills, ESCAPE);
 }
 
-float Client::CalcPriceMod(Mob* other, bool reverse)
+float Client::CalcClassicPriceMod(Mob* other, bool reverse) {
+	float priceMult = 0.8f;
```

Comment:
`price_multiplier` instead of `priceMult`.

Reply:
Fixed